hw1/libs/patch.js

```diff
@@ -0,0 +1,15 @@
+const { COLOR } = require('./env');
+const chalk = require('chalk');
+const log = console.log;
+
+global.console.log = (...args) => {
+    log(chalk[COLOR]("==="), chalk[COLOR](...args), chalk[COLOR]("==="))
+}
+
+global.console.short = (...args) => {
+    if(args.length > 2){
+        log(args[0], chalk[COLOR]("..."), args[args.length - 1])
```

Comment:
```suggestion
        console.log(args[0], "...", args[args.length - 1]);
```
console.log уже пропатченный - будет выводить данные в нужном цвете.
Не забывай ставить `;` в конце строки.